src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_NUMBER = 5;
+    private static final int MAX_NUMBER = 10;
+    private Random random = new Random();
```

Comment:
create a ColorSupplier field here as well and call its methods through this field